src/systems/ec/implementations/react/templates/pages/person-list/src/PersonListPage.jsx

```diff
@@ -0,0 +1,527 @@
+    )}
+
+    <div className="ecl-u-pv-xl ecl-u-pv-lg-3xl">
+      <div className="ecl-container">
+        <div className="ecl-row">
+          <div className="ecl-col-12 ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-3xl">
+            <h2 className="ecl-u-type-heading-2 ecl-u-type-color-black ecl-u-mt-none ecl-u-mb-l">
+              Contact the team
+            </h2>
+            <DescriptionList
+              variant="horizontal"
+              className="ecl-u-mt-none ecl-u-mb-l ecl-u-mb-lg-l"
+            >
+              <DescriptionTerm>Email</DescriptionTerm>
+              <DescriptionDefinition>
+                <a
+                  className="ecl-link"
+                  href="mailto:cab-andriukatis-webpage@ec.europa.eu"
+                >
+                  cab-andriukatis-webpage@ec.europa.eu
+                </a>
+              </DescriptionDefinition>
+              <DescriptionTerm>Address</DescriptionTerm>
+              <DescriptionDefinition>
+                European Commission
+                <br />
+                Rue de la Loi / Wetstraat 200
+                <br />
+                1049 Brussels
+                <br />
+                Belgium
+              </DescriptionDefinition>
+            </DescriptionList>
+            <Link
+              className="ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-3xl"
+              variant="standalone"
+              label="Media enquiries"
+              href="/example"
+              icon={{
+                shape: 'ui--corner-arrow',
+                size: 'xs',
+                transform: 'rotate-90',
+              }}
+            />
+          </div>
+        </div>
+        <div className="ecl-row">
+          <aside className="ecl-col-12 ecl-col-lg-3">
+            <h2 className="ecl-u-type-heading-2 ecl-u-type-color-black ecl-u-d-lg-none ecl-u-mt-none ecl-u-mt-none ecl-u-mb-xl ecl-u-mb-lg-l">
+              People (11)
+            </h2>
+            <h3 className="ecl-u-type-heading-3 ecl-u-type-color-black ecl-u-mt-none ecl-u-mb-lg-l">
+              Filter by
+            </h3>
+            <form>
+              <TextInput
+                className="ecl-u-mb-m"
+                id="facet-name"
+                label="Name"
+                width="m"
+              />
+              <Select
+                id="facet-team"
+                groupClassName="ecl-u-mb-m"
+                label="Team"
+                width="m"
+                options={[]}
+              />
+              <Select
+                id="facet-portfolio"
+                groupClassName="ecl-u-mb-m"
+                label="Portfolio"
+                width="m"
+                options={[]}
+              />
+
+              <Button
+                className="ecl-u-mt-none"
+                label="Search"
+                variant="search"
+              />
+            </form>
+          </aside>
+          <main className="ecl-col-12 ecl-col-lg-9">
```

Comment:
Should be `ecl-col-lg-8`

Reply:
It's 9, please check `Person list desktop-spacings.png`

Reply:
on the person list mockup, the team members list is 8 columns